.cloudbuild/ci/os-compatibility-test.yaml

```diff
@@ -0,0 +1,23 @@
+timeout: 25m
+
+options:
+   machineType: E2_HIGHCPU_32
+
+steps:
+  - name: public.ecr.aws/gravitational/teleport-buildbox-centos7:teleport11
```

Comment:
just realize we are using `teleport11` . should we use `teleport9`? Totally missed in other backport

Reply:
Good point. I've fixed the tag in this image. I'll create a matching PR in v10 in a sec.

Reply:
v10 PR https://github.com/gravitational/teleport/pull/16177